environments/R_env.yaml

```diff
@@ -0,0 +1,17 @@
+# env file for running scDINO
+name: R_timelapse_env
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python=3.8
+  - pip
```

Comment:
Double checking: do you still need pip here?

Reply:
removed!